app/views/foreman_ansible/ansible_roles/_select_tab_content.html.erb

```diff
@@ -2,7 +2,10 @@
 <%= webpacked_plugins_css_for :foreman_ansible %>
 
 <div class='tab-pane' id='ansible_roles'>
-  <% roles = f.object.is_a?(Hostgroup) ? roles_attrs(f.object.inherited_and_own_ansible_roles) : roles_attrs(f.object.all_ansible_roles) %>
+  <% roles = f.object.is_a?(Hostgroup) ?
+     model_roles_attrs(f.object, HostgroupAnsibleRole, :hostgroup_id, :hostgroup_ansible_role_id) :
+     model_roles_attrs(f.object, HostAnsibleRole, :host_id, :host_ansible_role_id) %>
```

Comment:
Wouldn't it be more readable to hide the implementation detail to the method?
```suggestion
  <% roles = attributes_for_assigned_ansible_roles(f.object) %>
```

and then in the method we coul have at the beggining something like:
```ruby
 if model.is_a?(Hostgroup)
  assoc_class, foreign_key, assoc_key = HostgroupAnsibleRole, :hostgroup_id, :hostgroup_ansible_role_id
 else # only other option is Host
   assoc_class, foreign_key, assoc_key = HostAnsibleRole, :host_id, :host_ansible_role_id
 end
```

Or we could implement that at the model level, what would make sense to me.